src/fluree/http_api/components/http.clj

```diff
@@ -40,13 +41,20 @@
 (def Context
   (m/schema ::fql/context {:registry fql/registry}))
 
+(defn edn-req->json-req
+  [req]
+  (log/trace "edn-req->json-req got:" req)
+  (if (and (contains? req :id) (contains? req :graph))
+    (cond-> {"@id"    (:id req)
+             "@graph" (:graph req)}
+            (contains? req :context) (assoc "@context" (:context req)))
+    req))
```

Comment:
If I'm understanding this correctly, this means users can't use an aliased `@id` or `@graph` in the `/create` endpoint, right?

This is different from `/transact`, which expands top-level keys using any provided `@context` before looking for the id and graph keys: https://github.com/fluree/db/blob/main/src/fluree/db/api/transact.cljc#L33.  So those keys can be aliased in the top-level context if desired. All of that parsing work is done on the `db` side.